cmd/delete.go

```diff
@@ -37,12 +41,48 @@ func deleteAppGroup(group string, namespace string, appFilter string) error {
 	return nil
 }
 
+const warningText = `Warning!
+This action could delete some resources like PVs, which can be in use from another party`
+
+func delYN(in io.Reader) (bool, error) {
+	r := bufio.NewReader(in)
+	fmt.Println(warningText)
+	for {
+		fmt.Print("Do you want to continue [y/n]")
+		response, err := r.ReadString('\n')
+		if err != nil {
+			return false, err
+		}
+		switch {
+		case len(response) == 2 && byte(unicode.ToLower(rune(response[0]))) == 'y':
+			return true, nil
+
+		case len(response) == 2 && byte(unicode.ToLower(rune(response[0]))) == 'n':
+			return false, nil
+		}
+	}
+
+}
+
 // deleteCmd represents the delete command
 var deleteCmd = &cobra.Command{
 	Use:   "delete",
 	Short: "Delete app group or app",
 	Args:  cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
+
+		askUserToConfirm, _ := cmd.Flags().GetBool("asking-for-confirmation")
```

Comment:
This is not consistent

Reply:
Fixed in https://github.com/DataReply/korgi/pull/15/commits/6134baeef0124ccf7393306f50571ce4e5e3028c.
Please check the new PR https://github.com/DataReply/korgi/pull/15, because I am going to remove my commits from this PR, as it was not open by me.